src/main/steps/existing-application/content.ts

```diff
@@ -31,7 +55,10 @@ const en = ({ isDivorce, partner, required, existingCaseId }: CommonContent, isI
   };
 };
 
-const cy: typeof en = ({ isDivorce, partner, required, existingCaseId }: CommonContent, isInviteCaseJoint: boolean) => {
+const cy: typeof en = (
```

Comment:
could add a todo comment for Welsh translation

Reply:
I think we need a ticket rather than a TODO
I'll ask mention it to Sanchita and raise a ticket if needed

Reply:
https://tools.hmcts.net/jira/browse/NFDIV-2870